pkg/cmd/config.go

```diff
@@ -39,7 +39,7 @@ import (
 
 const (
 	// defaultKubernetesVersion is default Kubernetes version for the example configuration file
-	defaultKubernetesVersion = "1.14.1"
+	defaultKubernetesVersion = "1.16.0"
```

Comment:
This should not go in this PR.

Reply:
Why? This is just for the example config, i.e. `kubeone config print -f`.

Reply:
Because it is not related to the purpose of this PR.

Reply:
And even more so because that change should go together in a change which updates the README about supported versions, perhaps the occurrences of `1.14.1` in the testdata manifests, and more.